src/main/java/org/broadinstitute/consent/http/util/DarUtil.java

```diff
@@ -35,4 +36,15 @@ public static boolean requiresManualReview(Document dar) throws IOException {
         return !fieldsForManualReview.stream().
                 filter(field -> form.containsKey(field) && Boolean.valueOf(form.get(field).toString())).collect(Collectors.toList()).isEmpty();
     }
+
+    public static  List<Integer> getIntegerList(Document dar, String key) {
+        List<Integer> result = new ArrayList<>();
+        List datasets = dar.get(key, List.class);
+        for(Object value : datasets){
+            if(value instanceof Integer) {
+                result.add((Integer)value);
+            }
+        }
+        return result;
```

Comment:
I know that you know I will always favor a lambda for this :-) This could be a lot simpler:
```
        List<Object> datasets = dar.get(key, List.class);
        return datasets.
                stream().
                filter(Integer.class::isInstance).
                map(Integer.class::cast).
                collect(Collectors.toList());
```